src/richie/apps/core/templates/social-networks/course-badges.html

```diff
@@ -0,0 +1,16 @@
+{% load i18n %}{% spaceless %}
+{% blocktrans with page_title=page_title page_url=page_url asvar twitter_text %}Je viens de m'inscrire pour '{{ page_title }}' via @funmooc: {{ page_url }}{% endblocktrans %}
```

Comment:
The messages here need to be generic richie messages.

Reply:
How to do them generic ? From settings ? But so how to reach this settings from template ?

Also from course to blogpost, message in share badges are not the same.